apps/meeting/src/containers/MeetingForm/index.tsx

```diff
@@ -250,6 +253,14 @@ const MeetingForm: React.FC = () => {
         checked={keepLastFrameWhenPaused}
         onChange={toggleKeepLastFrameWhenPaused}
       />
+      <FormField
+        field={Checkbox}
+        label="Skip meeting join device selection"
+        value=""
+        checked={skipDeviceSelection}
+        onChange={toggleMeetingJoinDeviceSelection}
+        infoText="Please select the devices manually to successfully join a meeting"
```

Comment:
device selection is not required to be able to join a meeting. It should be spectator mode in this case.

Reply:
The only reason to mention this is to showcase the `skipDeviceSelection` flag, since we already have a spectator mode, this would be good to tell "Hey if you are skipping device selection at join, then you have to manually select the devices, otherwise there will be issues/no audio or fake device selection etc". Under flags docs, we explicitly mention that it would be builders responsibility to do device selection for a working meeting.